<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="product_route_tests.ProductRouteTest-20200420141837" tests="2" file="product_route_tests.py" time="0.007" timestamp="2020-04-20T14:18:38" failures="0" errors="0" skipped="0">
	<testcase classname="product_route_tests.ProductRouteTest" name="test_get_products_with_none_in_database" time="0.005" timestamp="2020-04-20T14:18:38" file="src\unittest\python\product_route_tests.py" line="28">
		<system-out><![CDATA[]]></system-out>
		<system-err><![CDATA[]]></system-err>
	</testcase>
	<testcase classname="product_route_tests.ProductRouteTest" name="test_get_products_with_one_in_database" time="0.002" timestamp="2020-04-20T14:18:38" file="src\unittest\python\product_route_tests.py" line="36">
		<system-out><![CDATA[]]></system-out>
		<system-err><![CDATA[]]></system-err>
	</testcase>
</testsuite>
